src/foraging_gui/Foraging.py

```diff
@@ -815,7 +815,7 @@ def _OpenBehaviorFolder(self):
         except Exception as e:
             logging.error(str(e))
             try:
-                AnimalFolder=os.path.join(self.default_saveFolder, self.Tower.currentText(),self.ID.text())
+                AnimalFolder=os.path.join(self.default_saveFolder, self.ID.text())
```

Comment:
I suggest we keep the original folder structure to contain the box so we can easily identify which box the session came from without opening the json file.

Reply:
Yeah ok. I will add the current box as the folder.